### PR TITLE
fixes airlock audio inconsistency

### LINKED ISSUE #32874: Airlock sounds are inconsistent
Since airlocks open and close faster now, the sounds are outdated. The airlock finishes opening/closing long before the sound finishes playing.

### PR DESCRIPTION
**Pull request by ghost**
fixes #32874

### CODE REVIEW

**Comment by Xhuis** [comment]
I think the new audio sounds odd, since it's obviously just a quick hit to speed up the tempo of the sound. I don't mind the closing audio so much, but I think the opening sound is noticeably lower in quality (not the technical sense, but the "how good it sounds" sense.)

**Comment by Qustinnus** [comment]
the door just closes too fast for it to sound amazing.

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -1091,7 +1091,7 @@
 		if(obj_flags & EMAGGED)
 			return FALSE
 		use_power(50)
-		playsound(src.loc, doorOpen, 30, 1)
+		playsound(src.loc, doorOpen, 30)
```
Comment (ShizCalev) [inline code/game/machinery/doors/airlock.dm]: just src

also why are you removing the frequency variance?

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/game/machinery/doors/airlock.dm (hunk 1 above)
Comment (Qustinnus) [reply code/game/machinery/doors/airlock.dm]: the open sound sounds terrible with pitch change.

**Review by ShizCalev — APPROVED** [review]

**Comment by Shadowflame909** [comment]
STOP BURPING INTO MY DOORS REEEEEEEEEEEEEEEEEE

**Pull request merged**